bundler/spec/dependabot/bundler/file_updater/gemfile_updater_spec.rb

```diff
@@ -246,12 +247,17 @@
           groups: [],
           source: {
             type: "git",
-            url: "http://github.com/gocardless/business"
+            url: "http://github.com/dependabot-fixtures/"\
+            "dependabot-test-ruby-package"
           }
         }]
       end
 
-      it { is_expected.to include "\"business\", \"~> 1.14.0\", git" }
+      it do
+        is_expected.to include(
+          "\"dependabot-test-ruby-package\", \"~> 1.0.0\", git"
```

Comment:
Not entirely sure but looks like this spec was expecting an update matching the new requirements
```suggestion
          "\"dependabot-test-ruby-package\", \"~> 1.1.0\", git"
```

Reply:
Or maybe `1.0.1` https://github.com/dependabot-fixtures/dependabot-test-ruby-package/releases/tag/v1.0.1

Reply:
Ah ^ is a typo in the tagname, my bad